### DISCUSSION TITLE
apps/GFTP (Solved)

### DISCUSSION THREAD

**Discussion by 01fiore** [mocaccinoOS/mocaccino#83] [General] [answered]
I use GFTP to tickle my demodulator, I know it’s not here where to ask but I lost the link, thanks.

With the Beta 5, I met no problem of very good work, bravo.

**Comment by joostruis** [comment, marked as answer]
You better request this in community-repository : https://github.com/mocaccinoOS/community-repository/issues

**Reply by 01fiore** [reply]
Merci.
